### DISCUSSION TITLE
MRT with Joy UI

### DISCUSSION THREAD

**Discussion by baki-kucukcakiroglu** [KevinVandy/material-react-table#894] [Ideas]
In my opinion, Joy UI is a pretty good library and will be widely used in a couple of years. Why don't we have MRT with Joy UI? This would give this community the competitive advantage of being the first extensive table library using Joy UI. I also don't think it will be hard to implement since most of the component definitions are the same for MUI and JUI.

**Comment by KevinVandy** [comment]
You're preaching to the choir. I already bought the domain https://joy-react-table.com/ months ago. I have wanted to start this project for a while now, but literally no one else has asked for it yet. You're the first one. In the meantime, I have kept improving material-react-table itself and its docs and examples so that when I start the joy ui fork, it has an even better place to start from.

I think I'd really want another couple of people who are interested in working on it for OSS before starting the project. It would take 2 months of all my free time to get it up and running probably, or at least, that's what happened last year when I made mantine-react-table.
